pydra/engine/tests/test_shelltask.py

```diff
@@ -2267,12 +2267,14 @@ def change_name(file):
     )
 
     # TODO: not sure why this has to be string
-    in_file = Path(os.path.dirname(os.path.abspath(__file__))) / "data" / "foo.nii"
+    in_file = (
+        Path(os.path.dirname(os.path.abspath(__file__))) / "data_tests" / "test.nii.gz"
+    )
```

Comment:
```suggestion
    in_file = Path(__file__).parent / "data_tests" / "test.nii.gz"
```

Reply:
@effigies - are you correcting black? ;-)

Reply:
I think mine is short enough to go back to one line.